sdk/python/setup.py

```diff
@@ -45,6 +45,9 @@
     "pyarrow<0.16.0,>=0.15.1",
     "numpy",
     "google",
+    "s3fs",
+    "boto3",
```

Comment:
shouldn't AWS dependencies be optional?

Reply:
Since GCP is not i figured we could as well bundle AWS deps too. Otherwise in each tutorial we'd have to tell people to install them separately. We'd have also have to build two docker images (GCP and AWS flavored) at least for the job service.